test/quick_test_helper.rb

```diff
@@ -1,7 +1,4 @@
-if ENV['RAILS_ENV']
-  def rails_require(*)
-  end
-else
+if ENV['RAILS_ENV'] == "test"
```

Comment:
@davetron5000 I tried reverting f9e7708181773c1219a5eb571ec6aba874863999, and the [build failed](https://travis-ci.org/stitchfix/resque-brain/builds/80707341).  Thoughts on keeping the refactored conditional?  Thanks!